sdk/trace/simple_span_processor_test.go

```diff
@@ -150,6 +151,36 @@ func TestSimpleSpanProcessorShutdownOnEndConcurrency(t *testing.T) {
 	<-done
 }
 
+func TestSimpleSpanProcessorShutdownOnEndRace(t *testing.T) {
+	exporter := &testExporter{}
+	ssp := sdktrace.NewSimpleSpanProcessor(exporter)
+	tp := basicTracerProvider(t)
+	tp.RegisterSpanProcessor(ssp)
+
+	var wg sync.WaitGroup
+	wg.Add(2)
+	go func() {
+		_, span := tp.Tracer("test").Start(context.Background(), "A")
+		defer span.End()
+		wg.Done()
```

Comment:
I would rather add `defer wg.Done()` in the beginning of the function (safer if the goroutine would panic)